src/Sylius/Bundle/CoreBundle/Controller/CheckoutController.php

```diff
@@ -194,11 +194,12 @@ public function finalizeAction(Request $request, OrderInterface $order)
     }
 
     /**
+     * @param int $id
+     *
      * @return Response
      */
-    public function thankYouAction()
+    public function thankYouAction($id)
```

Comment:
In addition, I'm thinking if such a page should have an expiry time.

At the very least make sure it cannot be indexed: here's what NOT to do!:
![derp-commerce--order-summary-pages-indexed](https://cloud.githubusercontent.com/assets/5972864/13699553/04cf3e7e-e773-11e5-9f81-5881200d55a3.png)